enterprise/reporting/reporting.go

```diff
@@ -397,6 +423,7 @@ func (r *HandleT) mainLoop(ctx context.Context, clientName string) {
 			continue
 		}
 
+		lastReportedAt = reportedAt
```

Comment:
Is it ok to update this if reports are empty or `ErrNoReports` (sql. ErrNoRows) is returned  ?

Reply:
Yes we should update `lastReportedAt`. we don't what to trigger the alert if there are no metric to send to reporting.

Reply:
If the reports are empty this line will no be executed.